src/components/SingleReview.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { fetchReviewById } from "../api";
+
+
+export default function SingleReview({id}) {
+
+    const [review, setReview] = useState([])
+    const [isLoading, setIsLoading] = useState(true)
+
+    useEffect(() => {
+        fetchReviewById(id)
+    .then(response => {
+        setReview(response)
+        setIsLoading(false)
+    })
+    }, [])
+
+    const date = new Date(review.created_at)
+    const postedAt = (date.toLocaleString());
+
+    if (isLoading) {
+        return <h2>Page is loading....</h2>
+    }
+
+    return (
+        <div>
```

Comment:
I would remove this <div> or use it as the <section> and then have line 27 as <article> instead